### PR TITLE
Tests and fixes

### PR DESCRIPTION
**Pull request by evlekht**
Updated a lot of camino executor tests, small fixes and cosmetic changes

### CODE REVIEW

File: vms/platformvm/txs/executor/camino_tx_executor.go [hunk 1]
```diff
@@ -1347,10 +1345,6 @@ func addCreds(tx *txs.Tx, creds []verify.Verifiable) {
 }
 
 func (e *CaminoStandardTxExecutor) AddressStateTx(tx *txs.AddressStateTx) error {
-	if err := locked.VerifyNoLocks(tx.Ins, tx.Outs); err != nil {
```
Comment (c4t-ag) [inline vms/platformvm/txs/executor/camino_tx_executor.go:1350]: Where is this one instead?

**Review by c4t-ag — CHANGES_REQUESTED** [review]

File: vms/platformvm/txs/builder/camino_builder.go [hunk 2]
```diff
@@ -506,7 +506,6 @@ func (b *caminoBuilder) NewRegisterNodeTx(
 	if err != nil {
 		return nil, err
 	}
-	sigIndices := in.(*secp256k1fx.TransferInput).SigIndices
```
Comment (c4t-ag) [inline vms/platformvm/txs/builder/camino_builder.go:509]: Pls. explain, thx!

File: vms/platformvm/txs/executor/camino_tx_executor.go (hunk 1 above)
Comment (evlekht) [reply vms/platformvm/txs/executor/camino_tx_executor.go:1350]: Its already present in syntactic - it was duplicated

File: vms/platformvm/txs/builder/camino_builder.go (hunk 2 above)
Comment (evlekht) [reply vms/platformvm/txs/builder/camino_builder.go:509]:
```Go
type TransferInput struct {
	Amt   uint64 `serialize:"true" json:"amount"`
	Input `serialize:"true"`
}

type Input struct {
	// This input consumes an output, which has an owner list.
	// This input will be spent with a list of signatures.
	// SignatureList[i] is the signature of OwnerList[i]
	SigIndices []uint32 `serialize:"true" json:"signatureIndices"`
}
```

**Review by c4t-ag — APPROVED** [review]

**Pull request merged**